flink-core/pom.xml

```diff
@@ -63,6 +63,19 @@ under the License.
 			<artifactId>guava</artifactId>
 			<version>${guava.version}</version>
 		</dependency>
+
+        <dependency>
+            <groupId>com.clearspring.analytics</groupId>
+            <artifactId>stream</artifactId>
+            <version>2.7.0</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>it.unimi.dsi</groupId>
+                    <artifactId>fastutil</artifactId>
```

Comment:
We have so far refrained from including fastutil, as it is a huge dependency, and one usually needs only a minimal subset of the functionality. Can you elaborate what this is needed for?